mu/virtual_environment.py

```diff
@@ -836,40 +861,23 @@ def install_baseline_packages(self):
         --upgrade is currently used with a thought to upgrade-releases of Mu.
         """
         logger.info("Installing baseline packages.")
-        logger.info(
-            "%s %s",
-            wheels_dirpath,
-            "exists" if os.path.isdir(wheels_dirpath) else "does not exist",
+        zipped_wheels_filepath = os.path.join(
+            wheels_dirpath, "%s.zip" % mu_version

```

Comment:
Do we need to scape any characters from `mu_version`? Probably not if it complies with semver, but would be good to be able to confirm that, specially for Windows.

Reply:
Good point. Not sure how hard to try on this. The __version__ marker is in our control (ie it's not user-entered) and we expect it just to be letters, numbers and dots. I don't think semver allows for, say, colons or qmarks. 

I'm also not sure if there is an `os.path` function which will check or normalise. `os.path.normpath` just deals with slashes & `..` elements afaik.

Reply:
Thinking a little further, we could just lift a semver regex from, eg, here: 

https://gist.github.com/jhorsman/62eeea161a13b80e39f5249281e17c39

and raise an exception if we don't match. That should be safe, I think

Reply:
Sounds good. We could also have a test that fails if `__version__` doesn't comply with semver.